packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js

```diff
@@ -67,14 +68,13 @@ export default function InputDatePicker(props) {
 	return (
 		<Date.Manager
 			value={props.value}
-			textInput={props.textInput}
```

Comment:
props.textInput is useless since date input value is only determined by prop.value